### PR TITLE
README:  include information about contribsys gem credentials for google-books

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

To help the next person who hits the problem I hit.  :-P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Make sure that:
 * You have added the public SSH key, often `~/.ssh/id_rsa.pub`, from your machine to [GitHub](https://github.com/settings/keys)
 * You have previously `ssh`-ed into all servers.
   * NOTE: If you are unsure about this, run `bin/check_ssh -e [qa|stage|prod]` and watch the output for any errors!
+* NOTE: if you run `check_cocina`, you may need to ensure that you have the contribsys gem credentials available to for google-books to install the sidekiq-pro gem locally (the credential is already on our deploy target VMs).  You can get the env variable name and value from shared_configs for google-books -- it's in the shared_configs README.
```
Comment (mjgiarlo) [inline README.md:15]: `gem credentials available to for google-books` looks like it has an extraneous word.

Comment (mjgiarlo) [inline README.md:15]: Specifically, the credentials are in shared_configs for google-books-prod (not in either stage or QA).

**Review by mjgiarlo — APPROVED** [review]
Suggested a couple minor wording tweaks. Approving now.

**Comment by ndushay** [comment]
Thanks for 👀, Mike - fixing then merging.

**Pull request merged**